client/src/pages/Search.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import avatar from "../assets/dashboard/avatar.svg";
+import { FaSearch } from "react-icons/fa";
+import SearchProfileCard from "../components/SearchProfileCard";
+
+function Search() {
+  return (
+    <main className="lg:text-xl lg:text-lg">
+      <div className=" md:w-3/4 lg:ml-8 flex lg:grid lg:grid-cols-12 lg:flex-row my-6 bg-cardBg drop-shadow-lg mx-4 rounded-lg p-3">
+        <label
+          htmlFor="search"
+          className="hidden lg:flex lg:justify-center lg:items-center lg:text-xl text-primary font-bold text-sm col-span-4"
+        >
+          The Search Start here
+        </label>
+        <form action="#" className="my-4 col-span-7 w-full">
+          <div className="flex">
+            <input
+              className="border-solid rounded-tl-lg p-2 rounded-bl-lg text-sm w-full"
```

Comment:
On my browser the border doesn't show up, I think it has to be just "border" in Tailwind and it should show up.

Reply:
yes, I'll add it.